lesson_03/ht_template/home_task_queries.sql

```diff
@@ -8,38 +8,122 @@
 Вивести кількість фільмів в кожній категорії.
 Результат відсортувати за спаданням.
 */
--- SQL code goes here...
-
 
+with cat_qtt as (
+    select
+    category_id,
+    count(distinct film_id) as films_qtt
+    from film_category
+    group by 1
+    order by 2 desc
+)
+select
+name,
+films_qtt
+from cat_qtt as cq
+join category as ct
+using(category_id);
 
 /*
 2.
 Вивести 10 акторів, чиї фільми брали на прокат найбільше.
 Результат відсортувати за спаданням.
 */
--- SQL code goes here...
 
 
+with films_rentals as (
+    select
+    rental_id,
+    i.inventory_id,
+    i.film_id,
+    actor_id
+    from rental as r
+    join inventory as i
+    on r.inventory_id = i.inventory_id
+    join film_actor as fa
+    on i.film_id = fa.film_id
+)
+select
+actor_id,
+CONCAT(last_name, ', ', first_name) AS actor_name,
+count(distinct  rental_id) as rentals
+from films_rentals as fr
+join actor as a
+using(actor_id)
+group by 1,2
```

Comment:
Using numbers instead of exact field names is not recommended. I will explain this on a lesson.